glide.yaml

```diff
@@ -91,7 +91,7 @@ import:
   - proto
 - package: kubevirt.io/containerized-data-importer
   repo:    https://github.com/kubevirt/containerized-data-importer.git
-  version: v1.3.0
+  version: 254749a63d335a4ec1ef919a971cab0b68cf4980
```

Comment:
Why do not use the tag?

Reply:
Based on the input from @mmazur, I shifted this to a hash from a tag due to the latest tag not having a fix for api definitions.

Reply:
Yes, a version with the correct api definitions makes it less of a hassle to develop python code against CDI. Hopefully 1.4.2 comes out soon, but for the time being it'd be great if after all this work what gets merged doesn't have known issues.